runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -4727,6 +4709,15 @@ private int calculateSubscribeFlags(
             return flags;
         }
 
+        private boolean validContent(
+            String topic,
+            OctetsFW payload)
+        {
+            final ValueValidator contentValueValidator = supplyValidator.apply(topic);
+            return contentValueValidator == null ||
```

Comment:
Should we introduce a `ValueValidator.NOP` to return from `supplyValidator` so we can avoid these `null` checks?

Reply:
we already have ValueValidator no op, null check here is redundant. removed it.